src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -1179,6 +1205,17 @@ SolidMechanicsLagrangianFEM::applySystemSolution( DofManager const & dofManager,
   } );
 }
 
+void SolidMechanicsLagrangianFEM::solveLinearSystem( DofManager const & dofManager,
+                                                     ParallelMatrix & matrix,
+                                                     ParallelVector & rhs,
+                                                     ParallelVector & solution )
+{
+  // Flip system sign to ensure matrix is positive definite
+  matrix.scale( -1.0 );
+  rhs.scale( -1.0 );
```

Comment:
I needed this at some point for multiscale to work (I'll have to double check to see if still required), but probably not a good idea to apply in the general case.
```suggestion
  // Flip system sign to ensure matrix is positive definite
  matrix.scale( -1.0 );
  rhs.scale( -1.0 );
```